src/cumulative_imputation_links.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+
+
+def get_cumulative_links(
+    dataframe,
+    forward_or_backward,
+    time_difference,
```

Comment:
`time_difference` is optional in other functions, doesn't necessarily need to be changed just highlighting this

Reply:
Thanks. I'll change this